hoomd/md/EvaluatorNormalizer.h

```diff
@@ -0,0 +1,26 @@
+// Maintainer: jproc
+#ifndef __PAIR_EVALUATOR_NORMALIZER_H__
+#define __PAIR_EVALUATOR_NORMALIZER_H__
+
+#include "hoomd/HOOMDMath.h"
+
+template<class evaluator> class Normalized : public evaluator
```

Comment:
After a lot of experimentation this was the way I went about applying a scalar normalization hopefully efficiently. It could alternatively be done by splitting into two perNeighbor calls (pre and post eval) but I decided this seemed cleaner.